src/pages/home/report/ReportActionContextMenuItem.js

```diff
@@ -12,9 +12,15 @@ import styles from '../../../styles/styles';
  *
  * @param {Boolean} [isHovered]
  * @param {Boolean} [isPressed]
+ * @param {Boolean} [isComplete]
  * @returns {String}
  */
-function getButtonState(isHovered = false, isPressed = false) {
+function getButtonState(isHovered = false, isPressed = false, isComplete = false) {
+    // We force this one over other states
```

Comment:
I don't think you need this comment

Reply:
Ok. I thought of adding this as the order is necessary.